plugins/link.md

```diff
@@ -113,6 +113,31 @@ tinymce.init({
 });
 ```
 
+### `link_force_external_targets`
+
+This option allows you to set whether TinyMCE should prepend a `http://` or `https://` prefix if the supplied link does not already contain a protocol prefix.
```

Comment:
Perhaps a short note about how this interacts with `link_assume_external_targets`?